src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +92,14 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(people -> people.getAge() >= fromAge
+                        && people.getAge() <= maleToAge
+                        && people.getSex() == People.Sex.MAN
+                        || people.getAge() >= fromAge
```

Comment:
`people.getAge() >= fromAge` is common to male and female.
try to evoid code duplication